### PR TITLE
Add analytics for Sublime

### PR DESCRIPTION
**Pull request by TsvetanMilanov**
We need to redirect the analytics data sent from Sublime to separate analytics project so we can keep track of how many users are using Telerik AppBuilder for Sublime Text.

### CODE REVIEW

File: lib/services/analytics-service.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import {AnalyticsServiceBase} from "../common/services/analytics-service-base";
+
+export class AnalyticsService extends AnalyticsServiceBase implements IAnalyticsService {
+	private static SUBLIME_ANALYTICS_CLIENT_NAME = "Sublime";
+	private static SUBLIME_ANALYTICS_PROJECT_KEY = "0e2863c76b76409fa7bc140a2b36a066";
+
+	constructor(protected $logger: ILogger,
+		protected $options: ICommonOptions,
```
Comment (rosen-vladimirov) [inline lib/services/analytics-service.ts]: this should be `IOptions`

File: lib/services/analytics-service.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+		$analyticsSettingsService: IAnalyticsSettingsService,
+		$progressIndicator: IProgressIndicator) {
+		super($logger, $options, $staticConfig, $errors, $prompter, $userSettingsService, $analyticsSettingsService, $progressIndicator);
+	}
+
+	public trackFeature(featureName: string): IFuture<void> {
+		return (() => {
+			if (this.$options.analyticsClient === AnalyticsService.SUBLIME_ANALYTICS_CLIENT_NAME) {
+				super.restartEqatecMonitor(AnalyticsService.SUBLIME_ANALYTICS_PROJECT_KEY).wait();
+				super.trackFeature(featureName).wait();
```
Comment (rosen-vladimirov) [inline lib/services/analytics-service.ts]: shouldn't this be outside of the `if`

**Comment by rosen-vladimirov** [comment]
:+1:

**Comment by TsvetanMilanov** [comment]
run ci

**Comment by TsvetanMilanov** [comment]
run ci

**Comment by TsvetanMilanov** [comment]
run ci

**Comment by TsvetanMilanov** [comment]
run ci

**Pull request merged**